### DISCUSSION TITLE
[ 功能建议 ]新增批量操作

### DISCUSSION THREAD

**Discussion by cslingjun** [hhyo/Archery#1578] [Ideas]
### 目前遇到的问题/使用障碍
SQL工单过多，DBA处理工单较繁琐
### 希望如何解决/实现它
通过批量处理简化DBA日常操作
### 其他信息
![image](https://user-images.githubusercontent.com/24772668/163723909-22a8ac36-5a47-4681-8a5f-19cb29cf215c.png)
@hhyo 是否需要？

**Comment by hhyo** [comment]
很有必要的功能，欢迎贡献。有个小建议，可以把操作按钮统一放在提交按钮右侧，命名为选择操作，采用dropdown的形式折叠起来

**Comment by cslingjun** [comment]
> 很有必要的功能，欢迎贡献。有个小建议，可以把操作按钮统一放在提交按钮右侧，命名为选择操作，采用dropdown的形式折叠起来

批量提交SQL,这个需要在实例配置的时候配上db_name，因为批量提交，然后页面没有提供库级别选择。这种使用场景还是较少的，是基于批量提交的实例的db是有相同表结构的
![image](https://user-images.githubusercontent.com/24772668/164698454-8244c0df-4629-4415-8ae2-31c79b86feaa.png)
其他的批量处理 ，是通用的功能了，整理下我再提个pr吧

**Reply by 18018593226lt-cloud** [reply]
你好，这个功能已经有了吗 没看到呢

**Comment by qinqi123456** [comment]
1.8.4 提交SQL已经支持实例配置的时候配上db_name。
（1）多租户场景强烈建议支持“批量提交SQL”。
（2）批量时针对一个语句，可以插入到多个实例，多个schema。

**Comment by hydgladiator** [comment]
批量执行SQL的，能否增加“按照先后顺序执行前一个再执行下一个”这种业务场景？

**Comment by RankRao** [comment]
这个【批量】相关的功能，有PR吗？最新版Archery-1.14.0没看到有这些功能呀？
